### PR TITLE
feat: add rating

### PR DESCRIPTION
**Pull request by Ronak-Mutha**
# Rating (for eCommerce)

image

### CODE REVIEW

File: docs/image.html [hunk 1]
```diff
@@ -151,14 +156,16 @@ <h2>Responsive images</h2>
...
                                         <img class="img-responsive br-3" src="../assets/images/avatar-image.jpg"
                                             alt="reponsive image with rounded borders" />
                                     </div>
                                     <div class="p-5">
-                                        <img class="img-responsive br-circle" src="../assets/images/avatar-image.jpg" alt="round reponsive avatar" />
+                                        <img class="img-responsive br-circle" src="../assets/images/avatar-image.jpg"
+                                            alt="round reponsive avatar" />
                                     </div>
                                 </div>
                                 <figure class="code-container">
```
Comment (samyakshah3008) [inline docs/image.html:171]: Good semantic html tags ✅

**Review by samyakshah3008 — APPROVED** [review]
You have really used proper semantic tags where necessary, clean code as well 
Small suggestion: Just add few comments in html file as well for better review experience 

Rest is all good ✅

**Review by shivampandey0 — APPROVED** [review]
- CSS variables used for colors and space units
- Semantic HTML tags used

Good to merge

**Comment by Ronak-Mutha** [comment]
Got 2 reviews. Merging to dev.

**Pull request merged**